client/webserver/site/src/html/forms.tmpl

```diff
@@ -544,3 +544,18 @@
   </div>
 </div>
 {{end}}
+
+{{define "toggleWalletStatusConfirm"}}
+<div class="form-closer hoverbg"><span class="ico-cross"></span></div>
+<div class="px-2 py-1 text-center position-relative fs22 sans-light">
+   [[[change_wallet_status]]]
```

Comment:
"Change Wallet Status" is a bit confusing.  Why not get rid of `change_wallet_status` and instead use `[[[disable_wallet]]]` or `[[[enable_wallet]]]` spans that get enabled or disabled like the message text below this?